docs/Acceptable-Formulae.md

```diff
@@ -68,6 +68,9 @@ Clang is the default C/C++ compiler on macOS (and has been for a long time). Sof
 ### Stuff that requires heavy manual pre/post-install intervention
 We're a package manager so we want to do things like resolve dependencies and set up applications for our users. If things require too much manual intervention then they aren't useful in a package manager.
 
+### Stuff that uses non-default MPI implementations and linear algebra libraries
```

Comment:
This is really specific to a small number of formulae and doesn't warrant this amount of text being added. We also check with this already with an `audit` and this document is for things we can't audit.

Reply:
@MikeMcQuaid Two points/questions that need resolving:

- There *is* an audit for BLAS/LAPACK, __*but*__ we cannot check *why* someone is linking to Accelerate since it provides more than just BLAS/LAPACK. Should we still remove the BLAS/LAPACK stuff here?
- I cannot find an audit for MPICH vs Open-MPI. Should one be created instead of adding text here?

Reply:
> * Should we still remove the BLAS/LAPACK stuff here?

Yup.

> * I cannot find an audit for MPICH vs Open-MPI. Should one be created instead of adding text here?

Yes, please.

Reply:
Removed changes to acceptable formulae